### PR TITLE
Analytics: New triggers

### PR DESCRIPTION
**Pull request by gblakeman**
Add tracking events for some UI-only interactions that are not automatically tracked:

* Footer next/previous button clicks: `track-footer-nextprev`
* Items list:*
  * Tray toggles: `track-tray-toggle`
  * Single item click: `track-tray-list-click`
  * Shift-click on item(s): `track-tray-list-shift-click`
  * Meta-click (command-click) on item: `track-tray-list-meta-click`
* Pill menus:**
  * Menu toggle: `track-pill-menu-toggle`
  * Keyboard navigation, next/previous: `track-pill-menu-key-nextprev`
  * Keyboard navigation, commit: `track-pill-menu-key-return`

----

*Undo button (`step-back`) and Remix button (`remix-all`) already tracked.
**Mouse commit clicks already tracked as `content-load-item`.

### CODE REVIEW

File: src/views/forms-controls/PillActionPrimary.svelte [hunk 1]
```diff
@@ -71,6 +73,9 @@
       isMenuPositioned = false;
     }
 
+    // fire analytics event
+    sendActionToMain('track-pill-menu-toggle', {});
```
Comment (joeyyang) [inline src/views/forms-controls/PillActionPrimary.svelte:77]: is it ok to send `null` here and throughout (i.e., `sendActionToMain('track-pill-menu-toggle')`)?

**Review by joeyyang — APPROVED** [review]
Looks good, just a small question!

**Review by lynnerang — APPROVED** [review]
LGTM!

File: src/views/forms-controls/PillActionPrimary.svelte (hunk 1 above)
Comment (gblakeman) [reply src/views/forms-controls/PillActionPrimary.svelte:77]: Good catch – I was too lazy to look it up. Seems like it’s entirely optional. I’ll remove it.

**Pull request merged**